pilot/pkg/model/authentication_test.go

```diff
@@ -189,10 +189,12 @@ func TestGetPoliciesForWorkload(t *testing.T) {
 						Namespace:        "foo",
 					},
 					Spec: &securityBeta.RequestAuthentication{
-						Selector: &selectorpb.WorkloadSelector{
-							MatchLabels: map[string]string{
-								"app":     "httpbin",
-								"version": "v1",
+						TypeSelector: &securityBeta.RequestAuthentication_Selector{
```

Comment:
I think this is actually a breaking API change; more info in https://github.com/istio/api/pull/2888#discussion_r1300433592